src/components/Conversation/EmptyMessageFeed.tsx

```diff
@@ -87,6 +95,40 @@ export default function EmptyMessageFeed({ onAddMemberClick }: Props) {
     )
 }
 
+function TokenGatedMessage() {
+    const { tokenAddress, tokenIds, tokenType, stakingEnabled } = useSelectedRoom() || {}
```

Comment:
You can use the `useCachedTokenGate(useSelectedRoomId())` hook. It'll give you `tokenIds` that's an array, even if empty, and `stakingEnabled` that's always a boolean.

Reply:
You'd also know right away if your room is gated `useCachedTokenGate` gives null if it ain't.